src/styles/modules/_m-radio-group-box.sass

```diff
@@ -0,0 +1,138 @@
++b('radio-box')
+  display: grid
+  grid-column-gap: 10px
+  list-style-type: none
+  padding: 0
+  width: 100%
+
+  +element('item')
+    @include rem-fallback('padding', 15px 10px)
+    margin: 0
+    +rem-fallback('margin-top', $space-xs)
+    position: relative
+    border: 1px solid color(agray, light)
+    border-radius: $radius-m
+
+  +element('icon')
+    display: block
+    +rem-fallback('height', 30px)
+    +rem-fallback('margin-bottom', 5px)
+    text-align: center
+
+    svg
+      +rem-fallback('max-height', 30px)
+
+  +element('button')
+    position: absolute
+    visibility: hidden
+    +rem-fallback('top', 10px)
+    +rem-fallback('right', 10px)
+
+    &:hover ~ .#{$prefix-component}radio-box__outside
+      border: 2px solid color(ablue, base)
+
+      &.#{$prefix-component}radio-box__outside--color-blue
+        border: 2px solid color(awhite, base)
+
+        &::before
+          background: color(awhite, base)
+
+    &:disabled+.#{$prefix-component}radio-box__label
+      color: color(agray, lighter)
+
+    &:disabled ~ .#{$prefix-component}radio-box__outside
+      &::before
+        background: transparent
+
+      &:hover
+        border: 2px solid color(ablue, base)
+
+      &.#{$prefix-component}radio-box__outside--color-blue,
+      &.#{$prefix-component}radio-box__outside--color-grey,
+      &.#{$prefix-component}radio-box__outside--color-ltgrey
+        &::before
+          background: transparent
+
+    &:disabled+.#{$prefix-component}radio-box__label--color-blue
```

Comment:
This is definitely some expert-level css right here

Reply:
Why Thanks :)